optee-teec/src/session.rs

```diff
@@ -40,16 +44,58 @@ pub enum ConnectionMethods {
     LoginGroupApplication,
 }
 
+// Sessions are documented as threadsafe in GP. Sessions with non-static lifetimes
+// cannot be send because we cannot ensure that their context lives long enough, but
+// sessions with 'static lifetime should be Send.
+//
+// A few ways to construct a session with static lifetime:
+// 1. With the "owned" feature, use an `OwnedSession`
```

Comment:
Can we have a `hello-world` example for multi-thread session? It would be great if the example can be tested on our CI